pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -59,7 +59,7 @@ elif command -v dnf >/dev/null 2>&1; then
 	if [ -n "${pkgs}" ]; then
 		dnf_install_flags="-y --setopt=install_weak_deps=False"
 		if grep -q "Oracle Linux Server release 8" /etc/system-release; then
-			dnf_install_flags="${dnf_install_flags} --enablerepo ol8_baseos_latest --enablerepo ol8_codeready_builder"
+			dnf_install_flags="${dnf_install_flags} --repo ol8_baseos_latest --repo ol8_codeready_builder"
```

Comment:
Could you add comment line to explain the difference?

Reply:
Done. Hopefully it's descriptive enough combined with the commit message.